playwright/tests/desk-tool/documentActions.spec.ts

```diff
@@ -0,0 +1,113 @@
+import {test, expect} from '@playwright/test'
+import {createUniqueDocument, DEFAULT_DATASET, deleteDocument} from '../../helpers'
+
+/* 
+  **NOTE**
+  In these tests, we are not testing input events, so to speed up the tests we will use use `page.fill()` instead of `page.type()` to populate the document title field 
+*/
+
+const DOC_TITLE = 'Document actions [Playwright]'
+const inputSelector = 'data-testid=input-title >> input'
+
+// The path of the network request we want to wait for
+const documentActionPath = (action) =>
+  `*/**/mutate/${DEFAULT_DATASET}?tag=sanity.studio.document.${action}*`
+
+// Helper to generate path to document created in a test
+const generateDocumentUri = (docId) =>
+  `/${DEFAULT_DATASET}/desk/input-ci;documentActionsCI;${docId}`
+
+// Helper to get the path to the document the client creates for a test
+async function getDocumentUri(_type: string, title?: string) {
+  const testDoc = await createUniqueDocument({_type, title})
+  return [generateDocumentUri(testDoc?._id), testDoc?._id]
+}
```

Comment:
Maybe this function name could be more clear about the fact that it creates a document? And maybe it could return an object so it is easier to know what the properties returned represent?

```suggestion
async function createDocument(_type: string, title?: string): Promise<{studioPath: string; documentId: string}> {
  const testDoc = await createUniqueDocument({_type, title})
  return {studioPath: generateDocumentUri(testDoc?._id), documentId: testDoc?._id]}
}
```